commitizen/git.py

```diff
@@ -288,22 +247,16 @@ def get_core_editor() -> str | None:
     return None
 
 
-def smart_open(*args, **kargs):
+def smart_open(*args, **kwargs):
     """Open a file with the EOL style determined from Git."""
-    return open(*args, newline=get_eol_style().get_eol_for_open(), **kargs)
+    return open(*args, newline=get_eol_for_open(), **kwargs)
 
 
 def _get_log_as_str_list(start: str | None, end: str, args: str) -> list[str]:
     """Get string representation of each log entry"""
     delimiter = "----------commit-delimiter----------"
     log_format: str = "%H%n%P%n%s%n%an%n%ae%n%b"
-    git_log_cmd = (
-        f"git -c log.showSignature=False log --pretty={log_format}{delimiter} {args}"
-    )
-    if start:
-        command = f"{git_log_cmd} {start}..{end}"
-    else:
-        command = f"{git_log_cmd} {end}"
+    command = f"git -c log.showSignature=False log --pretty={log_format}{delimiter} {args} {f'{start}..{end}' if start else end}"
```

Comment:
I am a 1-liner, f-string and ternary lover, but here I think nested f-string + ternary makes it difficult to read, why not split it on a 2nd line?
```suggestion
    command_range = f'{start}..{end}' if start else end
    command = f"git -c log.showSignature=False log --pretty={log_format}{delimiter} {args} {command_range}"
```

Reply:
+1